.github/pull_request_template.md

```diff
@@ -3,7 +3,7 @@
 
 **Checklist:**
 - [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 
-
+<!--
```

Comment:
Please remove the HTML comments.

Reply:
I put them here intentionally, so that authors can immediately see some information without clicking the link, but if they create the PR as is, these content won't be too long and distractive comparing to the original PR description.

Reply:
To clarify, I meant to remove the commented out content since it is not visible. Would that be ok?

Reply:
Hi @joeliedtke, sorry I didn't notice the comment here.

For clarification, when authors create PRs, they can see the commented out content here quickly.
And we only want authors to read these commented out content, because they are long and not related to the PR from reviewers' point of view.

Does that make sense?